teuthology/orchestra/daemon.py

```diff
@@ -108,6 +193,16 @@ def signal(self, sig, silent=False):
 
         :param sig: signal to send
         """
+        if self.use_init:
+            self.log.info("using init to send signal")
+            self.log.info("WARNING init may restart after kill signal")
```

Comment:
- More vague usage of the term "init"
- `log.info()` and 'WARNING' ?
- What exactly does "init may restart" mean?